x-pack/legacy/plugins/lens/server/routes/field_stats.ts

```diff
@@ -7,7 +7,7 @@
 import Boom from 'boom';
 import DateMath from '@elastic/datemath';
 import { schema } from '@kbn/config-schema';
-import { AggregationSearchResponse } from 'elasticsearch';
+import { BaseAggregationSearchResponse } from 'elasticsearch';
```

Comment:
I don't think these are well-named any more: It looks like the only difference is that one of these types contains total hits as an integer, and the other as an object, like was changed in 7.0.

Potential naming suggestion:

* AggregationSearchResponseWithInt
* AggregationSearchResponseWithObject

Alternatively, maybe you can export the Base type and allow users to combine the `hits` property themselves?

Reply:
That sounds reasonable to me. Maybe even: `AggregationSearchResponseWithTotalHitsAsInt` and `AggregationSearchResponseWithTotalHitsAsObject`

Slightly verbose but often better to be explicit about these differences.